ngraph/core/include/openvino/core/descriptor/tensor.hpp

```diff
@@ -40,6 +40,7 @@ class OPENVINO_API Tensor {
     void set_name(const std::string& name);
 
     const std::unordered_set<std::string>& get_names() const;
+    void reset_names();
```

Comment:
why `set_names` with empty set does not work here?

Reply:
Because I need to update the id of tensor in order to regenerate name